### DISCUSSION TITLE
Sites using SWR

### DISCUSSION THREAD

**Discussion by shuding** [vercel/swr#478] [General]
Feel free to share your company/organization/personal websites if you are using SWR!

Some examples:
- [Vercel](https://vercel.com)
- [Backstage](https://github.com/spotify/backstage) by Spotify
- [learn-anything.xyz](https://learn-anything.xyz)
- [Dokz](https://dokz.site)
- [Hyper](https://hyper.is)
- [Deno Docs](https://doc.deno.land)
- [Pont](https://github.com/alibaba/pont) by Alibaba

**Comment by sergiodxa** [comment]
Algolia uses it in an internal project https://twitter.com/frontstuff_io/status/1275686690234028032

**Comment by lxsmnsyc** [comment]
Lyon uses it for the main website: https://lyon.com.ph

**Comment by bjcarlson42** [comment]
I'm using it for most of my api requests... https://benjamincarlson.io
